docs/HowToDevelopThingsUsingSTDK.md

```diff
@@ -0,0 +1,256 @@
+# How to develop things using STDK based TizenRT   
+
+STDK is one of Develop Kit(STDK) for Smart Things Eco-system.   
+This guide shows an example how to enable STDK and develop things.   
+
+So now, it will explain how to develop Led Things of SmartThings and Control it through SmartThings Mobile app.   
+
+## Prerequisites   
+The goal of this guide is to execute a sample application on esp32 board. so this doc will describe an experience how to build and run a STDK App. For this example, It need a ESP-WROVER-KIT board.   
+    
+* Install packages.   
+  Below package are needed to build.   
+```shell
+  sudo apt-get install gcc git wget make libncurses-dev flex bison gperf \
+      python python-pip python-setuptools \
+      python-serial python-cryptography python-future python-pyparsing python-pyelftools 
+
+  pip install pyserial
+```
+* Board specific prerequisites [[Details]](../README.md#supported-board--emulator).   
+* Toolchain installation [[Details]](../README.md#environment-setup).   
+
+### Get TizenRT Source.   
+#### Clone TizenRT source code.   
+```shell
+git clone https://github.com/Samsung/TizenRT.git
+cd TizenRT
+TIZENRT_BASEDIR="$PWD"
+```
+#### Apply patch for STDK.   
++ Update submodule   
+```shell
+git submodule update --init "external/stdk/st-device-sdk-c"
+git submodule update --init "external/libsodium/libsodium"
+```
++ Apply patch for stdk   
+```shell
+cd external/stdk/st-device-sdk-c
+git apply ../patch/0001-port-net-fix-ssl-issue-on-TizenRT.patch
+```
```

Comment:
A suggestion would be to replace above sections with a pointwise bullet list. As they are sequences, I think numbering them would be better. Please ignore if you think sections are more apt.

Reply:
Has Changed.